test/resources/awss3_jl_test.yaml

```diff
@@ -40,43 +45,509 @@ Resources:
                     - !Sub repo:${GitHubOrg}/${GitHubRepo}:ref:refs/heads/trying
                     - !Sub repo:${GitHubOrg}/${GitHubRepo}:ref:refs/heads/staging
                     - !Sub repo:${GitHubOrg}/${GitHubRepo}:ref:refs/heads/master
+          - Effect: Allow
+            Principal:
+              AWS: !Sub arn:aws:iam::${AWS::AccountId}:root
+            Action: sts:AssumeRole
```

Comment:
I've been using this for testing locally but I'd like to lock down further who can assume this role. I'll probably just remove this before merging this.

Reply:
I think this basically only lets admins AssumeRole, since it doesn't grant `sts:AssumeRole` permissions to any IAM identities explicitly